src/main/scala-2.11/org/apache/james/gatling/jmap/JmapAuthentication.scala

```diff
@@ -0,0 +1,50 @@
+package org.apache.james.gatling.jmap
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.concurrent.duration._
+
+class JmapAuthentication extends Simulation {
+
+  val domain = "domain-jmapauthentication.tld"
+  val username = "username@" + domain
+  val password = "password"
+
+  val httpProtocol = http
+    .baseURL("http://127.0.0.1")
+    .acceptHeader("application/json")
+    .contentTypeHeader("application/json; charset=UTF-8")
+
+  def obtainContinuationToken(username: String) = exec(
+    http("obtainContinuationToken")
+      .post("/authentication")
+      .body(StringBody(s"""{"username": "$username",
+        "clientName": "Mozilla Thunderbird",
+        "clientVersion": "42.0",
+        "deviceName": "Joe Blogg’s iPhone"}"""))
+      .check(status.is(200))
+      .check(jsonPath("$.continuationToken").saveAs("continuationToken")))
+
+  def obtainAccessToken(password: String) = exec(
+    http("obtainAccessToken")
+      .post("/authentication")
+      .body(StringBody(s"""{"token": "$${continuationToken}",
+        "method": "password",
+        "password": "$password"}"""))
```

Comment:
No :-)
$password reads var password (string substitution)
} closes the JSON here